docs/_getting-started-macos-ios.md

```diff
@@ -81,6 +81,14 @@ npx react-native init AwesomeTSProject --template react-native-template-typescri
 
 > **Note** If the above command is failing, you may have old version of `react-native` or `react-native-cli` installed globally on your pc. Try uninstalling the cli and run the cli using `npx`.
 
+### [Optional] Configuring your environment
+
+Starting from React Native version 0.69, it is possible to configure the Xcode environment using the `.xcode.env` file provided by the template.
+
+The `.xcode.env` file contains an environment variable to export the path to the `node` executable in the `NODE_BINARY` variable. You can customize this variable with your own path or your own node version manager.
+
+On top of this, it's possible to add any other environment variable and to source the `.xcode.env` file in your build script phases.
```

Comment:
I feel that here we're missing the point if this is needed or not.

Like from a user perspective, reading this I would question myself: "should I do this or not?". It's not clear if this is required or suggested. What are the implications of doing/not-doing this.

I'm not saying that we need to explain how this works here, but being more intentional is either suggesting the user to do this or not.

Reply:
Yup sounds great to me 👍